.actions/setup_tools.py

```diff
@@ -41,33 +43,56 @@ def _load_py_module(name: str, location: str) -> ModuleType:
     return py
 
 
-def load_requirements(
-    path_dir: str, file_name: str = "base.txt", comment_char: str = "#", unfreeze: bool = True
-) -> List[str]:
+class _RequirementWithComment(pkg_resources.Requirement):
```

Comment:
I need to pull these changes into a separate PR. They are necessary here because the load_requirements code breaks with `--extra-index-url https://download.pytorch.org/whl/cpu`